dev/com.ibm.websphere.appserver.features/visibility/public/jcacheApi-1.1/io.openliberty.jcacheApi-1.1.feature

```diff
@@ -1,10 +1,10 @@
 -include= ~${workspace}/cnf/resources/bnd/feature.props
-symbolicName = io.openliberty.jcache.autoapi-1.1
-visibility = private
-IBM-Provision-Capability:\
-  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=io.openliberty.jcache.internal-1.1))",\
-  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=io.openliberty.noShip-1.0))"
-IBM-Install-Policy: when-satisfied
+symbolicName=io.openliberty.jcacheApi-1.1
+visibility=public
+singleton=true
+IBM-ShortName: jcacheApi-1.1
+Subsystem-Version: 1.1
+Subsystem-Name: Java Cache (JCache) API 1.1
```

Comment:
This really bothers me as the public feature that the customer will enable for beta.  I think a better name should be chosen as we have discussed.

Reply:
Still waiting on discussion.

Reply:
We decided to rename to distributedSecurityCache-1.0.